app/src/main/kotlin/com/wire/android/ui/home/conversations/details/participants/GroupConversationParticipants.kt

```diff
@@ -102,11 +124,51 @@ fun GroupConversationParticipants(
                     )
                 }
             }
+        }
     }
 }
 
-@Preview
+@Composable
+fun MLSProgressIndicator(
+    progress: Float,
+    modifier: Modifier = Modifier,
+    color: Color = MaterialTheme.wireColorScheme.primary,
+    trackColor: Color = MaterialTheme.wireColorScheme.uncheckedColor
+) {
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .clip(RoundedCornerShape(dimensions().spacing24x)) // Add rounded corners here
+            .height(dimensions().spacing24x)
```

Comment:
issue: it's missing `contentAlignment = Alignment.Center,` and probably this height is too small or the font is a little too big because `Text` that's inside is taller than 24.dp, so even if I used `contentAlignment = Alignment.Center,`, it wasn't fully centered.

![image](https://github.com/wireapp/wire-android-reloaded/assets/30429749/d287f6e6-9e55-40a5-902f-b8ec6faa312c)

Can you share the link to the design so that we could compare it?

Reply:
> Can you share the link to the design so that we could compare it?

There's no design. It's a debugging tool for us only. It won't show in production.

However, it annoys me a bit the fact that it isn't centred 😅

Reply:
Sharp 👁️